### PR TITLE
docs(readme): Enhance "Necessary Changes" section of `README.md`

### PR DESCRIPTION
**Pull request by pklaschka**
This enhances the installation instructions, including info I didn't know when setting up a repo based on the template.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,9 +7,9 @@ Dependencies to the [main repo](https://github.com/TelestionTeam/telestion) are
 ## Necessary Changes
 
 * [ ] `settings.gradle` - set `rootProject.name`
-* [ ] `Dockerfile` - replace `TelestionTemplate` in the CMD block by the new project name 
-* [ ] `build.gradle` - replace `group` by your group name
-* [ ] `.github/workflows/publishImage.yml` - update `ORGANIZATION_OR_USER_NAME/REPO_NAME/IMAGE_NAME`
+* [ ] `Dockerfile` - replace `TelestionTemplate` in the CMD block by the new project's name (they must match exactly!)
+* [ ] `build.gradle` - replace `group` by your group name (it is recommended to follow Java [package naming conventions](https://docs.oracle.com/javase/tutorial/java/package/namingpkgs.html), i.e., beginning with the company url in reverse)
```
Comment (jvpichowski) [inline README.md:11]: Maybe you could add a link to group naming conventions in maven central and not only package naming conventions

File: README.md [hunk 2]
```diff
@@ -7,9 +7,9 @@ Dependencies to the [main repo](https://github.com/TelestionTeam/telestion) are
...
+* [ ] `.github/workflows/publishImage.yml` - update `ORGANIZATION_OR_USER_NAME/REPO_NAME/IMAGE_NAME`. Here, `IMAGE_NAME` doesn't have to match the project's name, but has to follow the rules for Docker Image names, cf. https://docs.docker.com/docker-hub/repos/#creating-repositories.
```
Comment (jvpichowski) [inline README.md:12]: I prefer a link to the github docker tutorial because it describes the naming scheme more precisely

**Review by jvpichowski — CHANGES_REQUESTED** [review]
I like the contribution but I think the links could be misleading

File: README.md (hunk 1 above)
Comment (pklaschka) [reply README.md:11]: Good idea, I'll look that up 🙂

File: README.md (hunk 2 above)
Comment (pklaschka) [reply README.md:12]: I'd love to, but I didn't find naming rules within GitHub... Do you, by any chance, have a link to that?

File: README.md (hunk 1 above)
Comment (pklaschka) [reply README.md:11]:
```suggestion
* [ ] `build.gradle` - replace `group` by your group name (it is recommended to follow [Maven Central `groupId` naming conventions](https://maven.apache.org/guides/mini/guide-naming-conventions.html), i.e., beginning with the company url in reverse)
```

**Pull request merged**